mlflow/cli.py

```diff
@@ -317,6 +317,15 @@ def _validate_static_prefix(ctx, param, value):  # pylint: disable=unused-argume
     "Default: Within file store, if a file:/ URI is provided. If a sql backend is"
     " used, then this option is required.",
 )
+@click.option(
+    "--artifacts-destination",
+    metavar="URI",
+    default="./mlartifacts",
+    help=(
+        "The base artifact location from which to resolve artifact upload/download/list requests "
```

Comment:
Love it! Can we also clarify that this only applies when the tracking server is configured to stream artifacts and the experiment artifact root is `http` or `mlflow-artifacts`?

Reply:
In general, we should figure out how to tell users about the `http` and `mlflow-artifacts` URIs for `--default-artifact-root` as well.

Reply:
Can we address this in a follow-up that adds support for `mlflow-artifacts` scheme?

Reply:
> In general, we should figure out how to tell users about the http and mlflow-artifacts URIs for --default-artifact-root as well.

That's a good idea. We should also make it easy for users to learn about these through the `mlflow server` CLI in a follow-up PR.